__tests__/app.test.js

```diff
@@ -47,3 +47,40 @@ describe("test endpoint /api", () => {
       });
   });
 });
+
+describe("tests endpoint /api/articles/:article_id", () => {
+  test("GET: 200 responds with the correct article object, with the correct properties", () => {
+    return request(app)
+      .get("/api/articles/2")
+      .expect(200)
+      .then(({ body: { article } }) => {
+        expect(article.article_id).toBe(2);
+        expect(article).toMatchObject({
+          article_id: expect.any(Number),
+          title: expect.any(String),
+          topic: expect.any(String),
+          author: expect.any(String),
+          body: expect.any(String),
+          created_at: expect.any(String),
+          votes: expect.any(Number),
+          article_img_url: expect.any(String),
+        });
```

Comment:
Since we know exactly which article we have requested on this endpoint, you could specify precisely what you expect all these values to be (as you have done with article_id on line 57). This would make the test very robust.
You've checked article_id twice in this test, which looks like a copy paste error